.cirrus.yml

```diff
@@ -497,7 +497,6 @@ task:
     - bin/flutter update-packages
   matrix:
     - name: tests_widgets-macos
-      only_if: $CIRRUS_BRANCH == 'master'
```

Comment:
Idk if we can enable these 2 shards for presubmit testing. WDYT @dnfield ?

Reply:
These were disabled for presubmits because it was hard to schedule the bots reliably. We can try enabling them again and disable them if it goes bad.  /cc @jonahwilliams

Reply:
SGTM!

Reply:
We should not need to schedule widget tests on PRs which do not touch the framework. I would not have expected cirrus scheduling to have improved in the meantime, so this should stay disabled.

Is it necessary for skia gold to run on presubmit for every PR?

Reply:
Ok. Thanks for the feedback! 
We can leave them disabled, but it would just come with the caveat that we wouldn't be testing/catching potential changes to golden files on mac. 
If a golden change does land on master though, it should show up for triage on the dashboard after post-submit checks. We just won't know until after post-submit testing is complete.

Reply:
WDYT of a future change where golden file tests run on their own shard? Right now they are mixed in with all of the other framework tests which is why all of them need to be executed.

Reply:
These tests could also be updated with a similar skip clause to the other framework tests in the meantime. Then they would still run on engine rolls or framework changes